index.js

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+//require("compareHTML_linebyline");
+var checker = require('./checker');
+const exec = require('child_process').exec;
+
+var fs = require("fs");
+var path = require("path");
+var program = require('commander');
+
+program
+	.arguments('<originalHTML>', 'Relative location of the Original HTML file to be compared.')
+	.arguments('<reproducedHTML>', 'Relative location of the Reproduced HTML file to be compared.')
+
+	.option('-o, --output <outputPath>', 'Desired Output location.')
+
+	.usage('[options] <originalHTML> <reproducedHTML>')
+
+	.action(function (originalHTML, reproducedHTML, program) {
+
+		var pathOriginalHTML = originalHTML,
+			pathReproducedHTML = reproducedHTML,
+			brokenPath = false;
+
+		var outputName = program.output;
+
+		try {
+			if (path.isAbsolute(originalHTML)) {
+				originalFileExisting = fs.statSync(originalHTML);
+			} else {
+				originalFileExisting = fs.statSync(path.join(process.cwd(), originalHTML));
+				pathOriginalHTML = path.join(process.cwd(), originalHTML);
+			}
+		}
+		catch (e) {
+			console.log("The path to your Original HTML file is invalid. Please check if the file exists.");
+			brokenPath = true;
+			return 404;
```

Comment:
This is not a web application, returning HTML error codes does not make sense.
Please check if [exit codes](http://tldp.org/LDP/abs/html/exitcodes.html) are more useful here.